app/controllers/InstanceRegistryController.scala

```diff
@@ -80,8 +89,16 @@ class InstanceRegistryController @Inject()(implicit system: ActorSystem, mat: Ma
     }
   }
 
+  /**Called to fetch network graph of current registry. Contains a list of all instances and all links
+    * currently registered.
+    *
+    * @return
+    */
+
   def getNetwork(): Action[AnyContent] = Action.async {
-    ws.url(instanceRegistryUri + "/network").get().map { response =>
+
+    ws.url(instanceRegistryUri + "/network").withHttpHeaders(authheader)
```

Comment:
I believe the api endpoint for the network request has changed on the instance registry. If I execute this branch this request fails. I will have a look into this when investigating the error you described in https://github.com/delphi-hub/delphi-management/issues/113 . Even though there you mention that the network() feature works.